components/FeedList/renderSwitchValue.js

```diff
@@ -0,0 +1,33 @@
+import SkeletonPosts from '../Skeleton/SkeletonPosts';
+import FeedItem from './FeedItem';
+import SkeletonPhotos from '../Skeleton/SkeletonPhotos';
+import Photos from '../../screens/ProfileScreen/page/Photos/Photos';
+
+const renderSwitchValue = (
+  item,
+  index,
+  navigation,
+  loading,
+  showPostsOrPhotos,
+  route,
+) => {
+  const renderPosts = () => {
+    return loading ? (
+      <SkeletonPosts />
+    ) : (
+      <FeedItem item={item} itemIndex={index} navigation={navigation} />
+    );
+  };
+
+  const renderPhotos = () => {
```

Comment:
const renderPhotos = () => loading ? 
  <SkeletonPhotos /> : <Photos item={item} />

Reply:
Done, thank you !